src/meta-srv/src/bootstrap.rs

```diff
@@ -229,8 +228,8 @@ pub async fn metasrv_builder(
         }
         #[cfg(feature = "pg_kvbackend")]
         (None, BackendImpl::PostgresStore) => {
-            let pg_client = create_postgres_client(opts).await?;
-            let kv_backend = PgStore::with_pg_client(pg_client)
+            let pool = create_postgres_pool(opts).await?;
+            let kv_backend = PgStore::with_pg_pool(pool)
                 .await
                 .context(error::KvBackendSnafu)?;
             let election_client = create_postgres_client(opts).await?;
```

Comment:
```suggestion
            // Client for election should be created separately since we need a different session keep-alive idle time.
            let election_client = create_postgres_client(opts).await?;
```